src/client/src/css/JobPosting.css

```diff
@@ -0,0 +1,105 @@
+.Jobpostingform {
+  padding: 10px;
+  display: grid;
+  align-items: center;
+  justify-content: center;
+  margin-top: 5%;
+  scroll-behavior: smooth;
+  overflow-y: scroll;
+  margin-bottom: 80px;
+}
+
+.title {
+  align-items: center;
+  display: flex;
+  justify-content: space-around;
+  margin: 0px auto;
+  margin-left: 0%;
+  margin-right: -18%;
+}
```

Comment:
We should add a margin for all input boxes.
I suggest the following: `margin: 10px auto`

Reply:
I agree but to keep the UI good I am keeping `margin: 5px auto`